genesyscloud/resource_genesyscloud_routing_email_route.go

```diff
@@ -36,35 +36,38 @@ func getAllRoutingEmailRoutes(_ context.Context, clientConfig *platformclientv2.
 	resources := make(ResourceIDMetaMap)
 	routingAPI := platformclientv2.NewRoutingApiWithConfig(clientConfig)
 
-	domains, _, getErr := routingAPI.GetRoutingEmailDomains(false)
-	if getErr != nil {
-		return nil, diag.Errorf("Failed to get routing email domains: %v", getErr)
-	}
+	for pageNum := 1; ; pageNum++ {
+		const pageSize = 100
+		domains, _, getErr := routingAPI.GetRoutingEmailDomains(pageNum, pageSize, false)
+		if getErr != nil {
+			return nil, diag.Errorf("Failed to get routing email domains: %v", getErr)
+		}
 
-	if domains.Entities == nil || len(*domains.Entities) == 0 {
-		return resources, nil
-	}
+		if domains.Entities == nil || len(*domains.Entities) == 0 {
+			return resources, nil
+		}
 
-	for _, domain := range *domains.Entities {
-		for pageNum := 1; ; pageNum++ {
-			const pageSize = 100
-			routes, resp, getErr := routingAPI.GetRoutingEmailDomainRoutes(*domain.Id, pageSize, pageNum, "")
-			if getErr != nil {
-				if isStatus404(resp) {
-					// Domain not found
-					break
+		for _, domain := range *domains.Entities {
```

Comment:
It looks like this code is the same code just formatted differently.  Make sure you run go fmt on your files if you have not done so